org.virt_manger.virt-viewer.json

```diff
@@ -0,0 +1,321 @@
+{
+    "app-id" : "org.virt_manager.virt-viewer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "virt-viewer",
+    "rename-desktop-file": "remote-viewer.desktop",
+    "rename-icon": "virt-viewer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--talk-name=org.a11y.*",
+        "--device=all",
+        "--filesystem=xdg-pictures",
+        "--filesystem=~/.ssh",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g"
+    },
+    "cleanup" : [
+        "/include",
+        "/sbin",
+        "/lib/pkgconfig",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/gir-*",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "eudev",
+            "config-opts": [
+                "--disable-hwdb",
+                "--disable-logging",
+                "--disable-gudev",
+                "--disable-introspection",
+                "--disable-keymap",
+                "--disable-mtd_probe"
+            ],
+            "cleanup": [
+                "/include",
+                "/etc",
+                "/libexec",
+                "/sbin",
+                "/lib/pkgconfig",
+                "/man",
+                "/share/aclocal",
+                "/share/doc",
+                "/share/gtk-doc",
+                "/share/man",
+                "/share/pkgconfig",
+                "*.la",
+                "*.a"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/gentoo/eudev/archive/v3.2.5.tar.gz",
```

Comment:
We have it in shared-modules: https://github.com/flathub/shared-modules/tree/master/udev

Reply:
I think the version in shared modules was out of date for my needs.